chart/templates/router/configmap.yaml

```diff
@@ -18,10 +18,12 @@ data:
   ROUTER_IMPORT_API_INTERNAL_URL: {{ include "carto.importApi.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}
   ROUTER_LDS_API_INTERNAL_URL: {{ include "carto.ldsApi.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}
   ROUTER_MAPS_API_INTERNAL_URL: {{ include "carto.mapsApi.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}
-  ROUTER_SSL_AUTOGENERATE: {{ .Values.customConfigValues.routerSslAutogenerate | quote }}
+  # If required, is going to be generated by Helm and inyected as secret
+  ROUTER_SSL_AUTOGENERATE: "0"
   ROUTER_SSL_CERTIFICATE_KEY_PATH: /usr/src/certs/{{ template "carto.tlsCerts.secretKeyKey" . }}
   ROUTER_SSL_CERTIFICATE_PATH: /usr/src/certs/{{ template "carto.tlsCerts.secretCertKey" . }}
   ROUTER_WORKSPACE_API_INTERNAL_URL: {{ include "carto.workspaceApi.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}
   ROUTER_WORKSPACE_WWW_INTERNAL_URL: {{ include "carto.workspaceWww.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}
   SELFHOSTED_DOMAIN: {{ .Values.customConfigValues.selfHostedDomain | quote }}
+  ROUTER_ENABLE_HTTPS: "true"
```

Comment:
shouldn't we put this as `false` is the LoadBalancer is going to do the TLS termination?

Reply:
Exact. For the moment I have left it static but when I document the ingress and certificates in the providers I will make it dynamic.